README.md

```diff
@@ -86,14 +105,12 @@ If you use GraphQL through sessions and CSRF, add the following to the `<head>`
 Modify the GraphQL UI config:
 
 ```diff
-React.createElement(GraphiQL, {
-    fetcher: GraphiQL.createFetcher({
-        url: '{{ $url }}',
-        subscriptionUrl: '{{ $subscriptionUrl }}',
-    }),
-+   defaultHeaders: JSON.stringify({
-+       'X-CSRF-TOKEN': document.querySelector('meta[name="csrf-token"]').content,
-+   }),
+const fetcher = GraphiQL.createFetcher({
+    url: '{{ $url }}',
+    subscriptionUrl: '{{ $subscriptionUrl }}',
++    headers: {
```

Comment:
I believe this will not allow ad-hoc modification of the headers anymore, which is why we chose `defaultHeaders` in the first place.

Reply:
See https://github.com/mll-lab/laravel-graphiql/issues/2#issuecomment-1333725321.